src/lib/SX127xDriver/SX127x.h

```diff
@@ -78,10 +77,11 @@ class SX127xDriver
     uint32_t GetCurrBandwidth();
     uint32_t GetCurrBandwidthNormalisedShifted();
 
-    void SetFrequency(uint32_t freq);
-    int32_t GetFrequencyError();
-    bool GetFrequencyErrorbool();
-    void SetPPMoffsetReg(int32_t offset);
+    void ICACHE_RAM_ATTR SetFrequency(uint32_t freq);
```

Comment:
You left a SetFrequency here.

Reply:
I didn't remove the old method completely, just added the SetFrequencyDirect()